clang_format/run_clang_format.sh

```diff
@@ -8,25 +8,17 @@ format_all() {
     CLANG_FORMAT_CONFIG=$(realpath $1)
 
     cd $BUILD_WORKSPACE_DIRECTORY
-    if ! test -f .clang-format; then
-        echo ".clang-format file not found. Bazel will copy the default .clang-format file."
-        cp $CLANG_FORMAT_CONFIG .
-    fi
     git ls-files '*.[ch]' '*.cpp' '*.cxx' '*.cc' '*.hpp' '*.hxx' \
-    | xargs -r $CLANG_FORMAT_BIN -i
+    | xargs -r $CLANG_FORMAT_BIN -i --style=file:$CLANG_FORMAT_CONFIG
```

Comment:
nit: [ShellCheck](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck) reports several issues in this script (missing double quotes around all variables, uses of backquotes), which might be worth fixing?

Reply:
Cool linter! I've fixed all the smells.